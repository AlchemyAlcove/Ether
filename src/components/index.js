import Button from "./button";
import Ellipsis from "./Ellipsis";

export{
  Button,
  Ellipsis
};
