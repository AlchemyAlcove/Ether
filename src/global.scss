$containerWidth: 1200px;
$desktopBreakpoint: 992px;
$headerHeight: 50px;
$tabletBreakpoint: 768px;

$background: #ECF0F1;
$headerBackground: #363D44;
$headerBackgroundHighlight: #24292D;
$headerDropdownBackground: #FFFFFF;
$headerLogoHighlight: #1EB6F2;
$headerText: #FFFFFF;
$headerTextOpaque: #BFC4CA;
$text: #333333;

$font: 'Open Sans', sans-serif;
$weightThin: 300;
$weightRegular: 400;
$weightThick: 700;

body, html {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
  background: $background;
  font-size: 14px;
  font-family: $font;
  font-weight: $weightRegular;
  color: $text;

  * {
    box-sizing: border-box;
  }
}

*:focus {
  outline: 0;
}

a {
  color: $text;
  text-decoration: none;
  cursor: pointer;
}

@import "./components/ellipsis/ellipsis.scss";