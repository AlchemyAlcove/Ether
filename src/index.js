import Ellipsis from "./components/ellipsis";
import Inject from "./utils/inject";

export{
  Ellipsis,
  Inject
};
