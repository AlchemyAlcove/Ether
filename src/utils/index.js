import setVariables from "./setVariables";

export{
  setVariables
};
