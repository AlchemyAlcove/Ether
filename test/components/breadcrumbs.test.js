import { Breadcrumbs, Crumb } from "../../src";
import { fireEvent, render } from "react-testing-library";
import React from "react"; // eslint-disable-line no-unused-vars

test("3 links", () => {
  const{container} = render(
    <Breadcrumbs>
      <Crumb>
        <a href="/pages">One</a>
      </Crumb>
      <Crumb className="funky">
        <a href="/pages/two">Two</a>
      </Crumb>
      <Crumb>Three</Crumb>
    </Breadcrumbs>
  );

  expect(container.firstChild).toMatchSnapshot();
});
