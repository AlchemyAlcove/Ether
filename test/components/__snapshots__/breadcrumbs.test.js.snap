// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`3 links 1`] = `
<nav
  class="breadcrumbs"
>
  <ol>
    <li
      class="crumb"
    >
      <a
        href="/pages"
      >
        One
      </a>
    </li>
    <li
      class="crumb funky"
    >
      <a
        href="/pages/two"
      >
        Two
      </a>
    </li>
    <li
      class="crumb"
    >
      Three
    </li>
  </ol>
</nav>
`;
